.github/workflows/lint-pr.yml

```
@@ -0,0 +1,18 @@
+name: 'Lint PR'
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      # Please look up the latest version from
+      # https://github.com/amannn/action-semantic-pull-request/releases
```

@raphaelschwinger Renovate will do that for us 😉 

```suggestion
      # https://github.com/amannn/action-semantic-pull-request/releases
```